index.js

```diff
@@ -9,23 +10,34 @@ module.exports = ({ logger = console, level = 'error', exposeStackTrace = false,
         if (filter(error) && typeof logger[level] === 'function') {
             logger[level](
                 {
-                    error: R.pick(['name', 'message', 'stack', 'details', 'status', 'statusCode', 'expose'], error),
+                    error: (({ name, message, stack, details, status, statusCode, expose }) => ({
+                        name,
+                        message,
+                        stack,
+                        details,
+                        status,
+                        statusCode,
+                        expose,
+                    }))(error),
                 },
-                `${R.prop('name', error)}: ${R.prop('message', error)}`
+                `${error.name ?? ''}: ${error.message ?? ''}`
             );
         }
 
         // eslint-disable-next-line no-param-reassign
         handler.response = {
-            ...R.propOr({}, 'response', handler),
-            statusCode: R.propOr(500, 'statusCode', error),
+            ...(handler.response ?? {}),
```

Comment:
You can spread `null` and `undefined` without fallback.
```suggestion
            ...handler.response,
```